packages/svelte2tsx/src/svelte2tsx/index.ts

```diff
@@ -251,9 +260,15 @@ function processSvelteTemplate(
         }
     };
 
-    convertHtmlxToJsx(str, htmlxAst, onHtmlxWalk, onHtmlxLeave, {
-        preserveAttributeCase: options?.namespace == 'foreign'
-    });
+    if (options.mode === 'ts') {
```

Comment:
The code transformations for the script tags stays mostly untouched, the htmlx2jsx transformation is completely now however.